### ISSUE TITLE
[Bug]: maps not shown under "my maps"

### ISSUE THREAD

**Issue by AdamRGrey** [nextcloud/maps#1146]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Maps app **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

when I make a custom map, a folder is created under /Maps for my user. :+1: good.
when I put stuff in that folder, it gets picked up for that map. :+1: good.
I get brought to a page with a static link that represents that map. :+1: good.

but the custom map doesn't show up under "my maps" on the right. I can figure out the static link from the url of the folder - a workaround for more tech savvy users - but not for most

### Steps to reproduce

1. click the + on the My Maps line
2. refresh the page

### Expected behavior

new map visible in my maps list

### Installation method

None

### Operating system

None

### PHP engine version

None

### Web server

None

### Database engine version

None

### Is this bug present after an update or on a fresh install of the server?

None

### Is this bug present after an update or on a fresh install of the app?

None

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### List of activated Apps

```shell
Enabled:
  - calendar: 4.2.3
  - circles: 25.0.0
  - cloud_federation_api: 1.8.0
  - comments: 1.15.0
  - contacts: 5.0.3
  - contactsinteraction: 1.6.0
  - dashboard: 7.5.0
  - dav: 1.24.0
  - deck: 1.8.3
  - federatedfilesharing: 1.15.0
  - federation: 1.15.0
  - files: 1.20.1
  - files_external: 1.17.0
  - files_pdfviewer: 2.6.0
  - files_rightclick: 1.4.0
  - files_sharing: 1.17.0
  - files_versions: 1.18.0
  - imageconverter: 1.3.4
  - logreader: 2.10.0
  - lookup_server_connector: 1.13.0
  - maps: 1.0.2
  - nextcloud_announcements: 1.14.0
  - notifications: 2.13.1
  - oauth2: 1.13.0
  - password_policy: 1.15.0
  - privacy: 1.9.0
  - provisioning_api: 1.15.0
  - serverinfo: 1.15.0
  - settings: 1.7.0
  - sharebymail: 1.15.0
  - systemtags: 1.15.0
  - tasks: 0.14.5
  - text: 3.6.0
  - theming: 2.0.1
  - twofactor_backupcodes: 1.14.0
  - updatenotification: 1.15.0
  - user_status: 1.5.0
  - viewer: 1.9.0
  - workflowengine: 2.7.0
Disabled:
  - activity: 2.17.0
  - admin_audit
  - bruteforcesettings
  - encryption
  - files_trashbin: 1.15.0
  - firstrunwizard: 2.14.0
  - photos: 2.0.1
  - recommendations: 1.4.0
  - related_resources: 1.0.3
  - support: 1.8.0
  - survey_client: 1.13.0
  - suspicious_login
  - twofactor_totp
  - user_ldap
  - weather_status: 1.5.0
```

**Comment by tacruc** [comment]
Can you try to register the maps mime types. 
php -f occ maps:register-mimetypes

(The command is only available in 1.2.0-1-nightly)

**Comment by tacruc** [comment]
Assume that fixed it.
